tools/misbehaving-jmx-server/src/main/java/org/datadog/supervisor/App.java

```diff
@@ -123,12 +122,16 @@ static void stopJMXServer() throws IOException, InterruptedException {
     }
 
     static void startJMXServer() throws IOException {
-        ProcessBuilder pb = new ProcessBuilder("java",
+        final String misbehavingOpts = System.getenv("MISBEHAVING_OPTS");
```

Comment:
Can you add something to the docs about this var and what format it should have?